src/main/java/mate/academy/repository/book/BookRepository.java

```diff
@@ -12,12 +12,8 @@
 public interface BookRepository extends JpaRepository<Book, Long>,
         JpaSpecificationExecutor<Book> {
 
-    Book save(Book book);
-
     @Query("SELECT b FROM Book b JOIN b.categories c WHERE c.id=:categoryId")
     List<Book> findAllByCategoryId(Long categoryId);
 
     Optional<Book> getBookById(Long id);
```

Comment:
do you really need your custom method for this operation? Please use OOTB methods of JpaRepository instead